engine/src/main/java/org/terasology/rendering/opengl/FBO.java

```diff
@@ -50,11 +55,22 @@
     private static final Logger logger = LoggerFactory.getLogger(FBO.class);
 
     public int fboId;
-    public int colorBufferTextureId;
-    public int depthStencilTextureId;
+    public enum BaseFboBuffer {
+        COLOR_TEXTURE("COLOR_TEXTURE"),
+        DEPTH_STENCIL_TEXTURE("DEPTH_STENCIL_TEXTURE"),
+        NORMAL_BUFFER_TEXTURE("NORMAL_BUFFER_TEXTURE"),
+        LIGHT_BUFFER_TEXTURE("LIGHT_BUFFER_TEXTURE");
+        String identifier;
+        private BaseFboBuffer(String identifier) {
```

Comment:
I feel pretty silly when I look at this. I guess that is why code review is good.